### ISSUE TITLE
Create README to describe project better

### ISSUE THREAD

**Issue by troy0820** [757colorcoded/server4ways#6]
**Is your feature request related to a problem? Please describe.**
The README for this repository lacks an understanding of what this is supposed to do.

**Describe the solution you'd like**
- Create a README that explains that server4ways is a learning tool designed to help people unfamiliar with git to create oss in different languages.

**Comment by gaurav-ag10** [comment]
can I help?

**Comment by troy0820** [comment]
Absolutely.  The premise of the repository is to create a server with 4 different ways.  Creating a README to explain will help others understand what that means.
